tests/scalapack/scalapack_05.mpirun=1.output

```diff
@@ -1,9 +1,9 @@
 32 32 1 1
 0.410521 0.471758
 64 32 1 1
-0.44694 0.478752
+0.44694 0.478753
```

Comment:
looks like this broke the test on all testers: 

https://cdash.kyomu.43-1.org/testSummary.php?project=1&name=scalapack%2Fscalapack_05.mpirun%3D1.debug&date=2018-01-24

https://cdash.kyomu.43-1.org/testDetails.php?test=66381565&build=14803

I think we should instead increase the number of significant digits in output.